.ci_support/linux_64_.yaml

```
@@ -1,8 +1,10 @@
 cdt_name:
-- cos7
+- conda
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
+- quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.9'
```

PyBIS still officially supports Python 3.6

See: https://sissource.ethz.ch/sispub/openbis/-/blob/a8a7d7601530bae9dba8a98246801f304d9f2c09/api-openbis-python3-pybis/src/python/setup.py

```suggestion
- '3.6'
```